ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -561,24 +563,13 @@ open class DrawerController: UIViewController, TokenizedControlInternal {
         return dismissPresentingViewController(animated: true)
     }
 
-    func updateCurrentTokens(_ tokens: DrawerTokens) {
-        self.tokens = tokens
-    }
+    public typealias TokenSetType = DrawerTokenSet
+    public typealias TokenSetKeyType = DrawerTokenSet.Tokens
+    public var tokenSet: DrawerTokenSet = .init()
 
-    var defaultTokens: DrawerTokens = .init()
-    var tokens: DrawerTokens = .init()
-    var overrideTokens: DrawerTokens? {
-        didSet {
-            updateDrawerTokens()
-        }
-    }
+    var overrideTokensSink: AnyCancellable?
```

Comment:
what does this mean?

Reply:
Ah, an old name from old PR. I'll update this.

Idea is that it's the Combine-y way of responding to changes to changes in `tokens`, allowing us to update the appearance whenever a token change is observed.